upgraderesponder/service.go

```diff
@@ -80,13 +81,14 @@ type CheckUpgradeRequest struct {
 }
 
 type CheckUpgradeResponse struct {
-	Versions []Version `json:"versions"`
+	Versions    []Version     `json:"versions"`
+	QueryPeriod time.Duration `json:"queryPeriod"`
```

Comment:
`time.Duration`? I am not sure how that would be passed back and how it can translate well across different languages of the client.

use an `int` instead. e.g. `requestIntervalInHours`

Same for the input parameter.

Also, `requestInterval` is better than `queryPeriod` as the response field back to the user. Because we're asking the client to send a request after x hours, not to ask the client to decide how the server should query the result.

Reply:
For the `time.Duration`, it is just `int64` in `ns`
```
{
    "versions": [
        {
            "Name": "v3.0.0",
            "ReleaseDate": "2020-05-18T12:30:00Z",
            "Tags": [
                "latest"
            ]
        },
    ],
    "queryPeriod": 3600000000000
}
```

But sure, having unit in the name make it easier to handle. I will fix it

Reply:
For  the input parameter, do you mean the `--query-period` flag? I don't want to hard-code it in `hour` because while doing development I want to set it to a few minutes

Reply:
ok that's fine.